### ISSUE TITLE
No option to view flight information once cxld

### ISSUE THREAD

**Issue by ali206** [206Tours/DB#1522]
![image](https://cloud.githubusercontent.com/assets/8440903/18890740/7b2eac44-84d0-11e6-97f8-be908c6ab868.png)

Even though a client may cxl, all of their ticketing information should remain in the DB

Ex: 423595 has 4 tickets (3 tckts & 1 ref) but nothing shows in the DB = not correct

![image](https://cloud.githubusercontent.com/assets/8440903/18890769/97c1ee3e-84d0-11e6-8365-837c389733aa.png)

![image](https://cloud.githubusercontent.com/assets/8440903/18890787/a8611436-84d0-11e6-994f-babb26eb4101.png)

**Comment by bradatkins** [comment]
sorry, why would we want this?

**Comment by chris206** [comment]
Maybe we could store the image for 206 Tours use upon cancelling? 
Otherwise, we may have no record of what flight they were on

**Comment by ali206** [comment]
for ex:

If someone cancels, part of their penalty is their airfare.

All costs should always remain in the DB so that if we ever need to look back or have questions, everything we need will be in the DB. 

I can always go check tickets in DRS but then to me that means the DB is not correctly storing all ticket information as I thought DB should be on the same level as ARC.

**Comment by ali206** [comment]
We should have more than an image, it should be the original ticket #/cost, then same ticket # showing a refund if any.  If we could not refund client's ticket than what would show in DB under that booking is orig flight #/cost

**Comment by bradatkins** [comment]
The costs will still be in the DB (eg in the Recap or ARC tickets).

The itinerary says nothing about costs though?

**Comment by ali206** [comment]
right, but the costs don't show in the DB and as you can see in my first post, there's no airplane button anymore to see the ticket #.

I can't remember so i'm assuming flight info was showing before they cxld - no longer in recap:

![image](https://cloud.githubusercontent.com/assets/8440903/19042853/bb724182-895b-11e6-992a-0abfa608b4a5.png)

And in ARC Tickets section in DB searching by 423595 doesn't pull any information up - so if I wasn't always double checking I would assume she never had an issued ticket:

![image](https://cloud.githubusercontent.com/assets/8440903/19042882/dcfd158e-895b-11e6-8b4c-fd542cd8ff2f.png)
